life/src/main.rs

```diff
@@ -0,0 +1,475 @@
+use macroquad::{color::{BLACK, LIGHTGRAY, RED, WHITE}, input::{get_last_key_pressed, is_mouse_button_down, mouse_position, KeyCode}, shapes::{draw_line, draw_rectangle}, text::draw_text, time::get_time, window::{clear_background, next_frame, Conf}};
+
+const ROWS: usize = 3;
```

Comment:
A suggestion if you want to make this even better (maybe out of scope for the project haha):

use the serde crate and the toml crate and put this as a config file, or use clap to pass it as cli arguments